GPflow/session.py

```diff
@@ -0,0 +1,77 @@
+import os
+import warnings
+import tensorflow as tf
+from tensorflow.python.client import timeline
+from ._settings import settings
+
+
+class TracerSession(tf.Session):
+    def __init__(self, *args, **kwargs):
```

Comment:
This won't allow `*args` to be passed to `tf.Session` I believe. Not a huge problem.